charts/grafana/templates/headless-service.yaml

```diff
@@ -15,4 +15,8 @@ spec:
   selector:
     {{- include "grafana.selectorLabels" . | nindent 4 }}
   type: ClusterIP
+  ports:
+  - protocol: TCP
+    port: 3000
+    targetPort: 3000
```

Comment:
Shouldn't this be set to `9094` so that Grafana can query `{{ Name }}-headless:9094` ?

Reply:
It doesn't really matter, this is only used for a normal deployment to bind to the service as otherwise it wouldn't if there is no port. But I can change it to 9094, so it reflects more the usage of the service.

Reply:
Ah, understood, that makes sense.

I'm still seeing an issue on a local copy of this. Will comment in main thread to discuss further